test.js

```diff
@@ -42,26 +41,65 @@ const bookkeepingConfigurations = {
   blockHeadersToSend: 27
 };
 
+/**
+ * Creates a fork object with the provided name and activation height.
+ * The fork object will have a method called `isAlreadyActive` to check if the fork is already active,
+ * which will get the latest block number and compare it with the `activationHeight` and cache the result to avoid calling the blockchain again
+ * if the fork is already active because a fork is activated only once.
+ * @param {string} name 
+ * @param {number} activationHeight 
+ * @returns {{name: string, activationHeight: number, isAlreadyActive: function(RskTransactionHelper): Promise<boolean>}}
+ */
+const createForkObject = (name, activationHeight) => {
+  // 'Private' variable to hold if the fork is already active
+  let isActive = false;
+
+  async function isAlreadyActive(rskTxHelper) {
+    // If the fork is already active, then return the cached result
+    if(isActive) {
+      return isActive;
+    }
+    if(this.activationHeight === -1) {
+      return false;
+    }
+    rskTxHelper = rskTxHelper || getRskTransactionHelper();
+    // Cache the result to avoid calling the network again if the fork is already active
+    const latestBlockNumber = await rskTxHelper.getBlockNumber();
+    isActive = latestBlockNumber >= this.activationHeight;
+    return isActive;
+  }
+
+  return {
+    name,
+    activationHeight,
+    isAlreadyActive,
+  };
+
+};
+
+
 // ***** GLOBALS ***** //
 global.Runners = {
   hosts: {},
   common: {
     forks: {
-      orchid: 1,
-      wasabi100: 1,
-      papyrus200: 1,
-      iris300: 1,
-      hop400: 1,
-      hop401: 1,
-      fingerroot500: 1,
-      arrowhead600: 1,
-      arrowhead631: 1,
-      lovell700: -1,
+      orchid: createForkObject('orchid', 1),
+      wasabi100: createForkObject('wasabi100', 1),
+      papyrus200: createForkObject('papyrus200', 1),
+      iris300: createForkObject('iris300', 1),
+      hop400: createForkObject('hop400', 1),
+      hop401: createForkObject('hop401', 1),
+      fingerroot500: createForkObject('fingerroot500', 1),
+      arrowhead600: createForkObject('arrowhead600', 1),
+      arrowhead631: createForkObject('arrowhead631', 1),
+      lovell700: createForkObject('lovell700', 1100),
     },
     additionalFederationAddresses: []
   }
 };
 
+Runners.config = config;
```

Comment:
Making the configuration (regtest.js or regtest-all-keyfiles.js, etc) global so we can access it anywhere.